src/main/kotlin/lotto/domin/LottoNumberSet.kt

```diff
@@ -0,0 +1,5 @@
+package lotto.domin
+
+data class LottoNumberSet(
```

Comment:
```suggestion
@JvmInline
value class LottoNumberSet(
```
값을 하나만 캡슐화하는 경우에는, value(inline) class를 활용할 수 있어요.
data class는 힙 영역을 사용하지만 inline class는 스택 영역을 사용하기 때문에, 메모리를 절약할 수 있고 속도 측면에서도 약간의 이점이 있습니다. (이하 유사 관련 내용 동일)

ref. https://quickbirdstudios.com/blog/kotlin-value-classes/